hieroplatform/src/main/java/org/hiero/sketch/spreadsheet/Bucket1D.java

```diff
@@ -0,0 +1,84 @@
+package org.hiero.sketch.spreadsheet;
+
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+/**
+ * A one dimensional bucket designed for a Histogram1D
+ */
+public class Bucket1D {
+    private Object minObject;
+    private Object maxObject;
+    private double minValue;
+    private double maxValue;
+    private long count;
+
+    public Bucket1D() { }
+
+    private Bucket1D(final long count, final double minValue,
```

Comment:
This constructor can start by calling the other one:
{ this(minValue, maxValue); ... }